### ISSUE TITLE
Collapsing folders feature on nav bar is problematic

### ISSUE THREAD

**Issue by apinnick** [theforeman/foreman-documentation#3531]
Can we please reverse the new nav bar feature where folders that are not in focus are collapsed? This feature makes it impossible for me to view the contents of 2 folders simultaneously or to review the overall structure of the document.

If there is a strong preference for collapsing folders, could we set up an alternative view that exposes the entire TOC?

**Comment by maximiliankolb** [comment]
Not sure if bad advice or a neat workaround: If you use a container to build foreman-documentation, then you won't see any navigation at the top or any JS applied to the navigation on the side. :upside_down_face: (it works for me)

On a more serious note: If it's solely about reviewing content, then maybe we can come up with a small hack or `make` target that builds the navigation? Or maybe a button (for everyone) that expands the complete nav bar?

**Comment by apinnick** [comment]
This is mainly for reviewing PRs, especially ones that involve major content restructuring.

However, the feature is just plain irritating. I want the choice to view more than one folder at the same time, without a script deciding what I can view. I don't want to see a navigation build unless it includes the content. An optional view with the complete nav bar would be OK.

Was this feature implemented after discussion and testing? I recall that Lena created a PR but I do not recall whether it was to resolve an issue.

**Comment by maximiliankolb** [comment]
> However, the feature is just plain irritating.

I like it!

> Was this feature implemented after discussion and testing? I recall that Lena created a PR but I do not recall whether it was to resolve an issue.

It was tested and reviewed, similar to every other upstream PR.

> I don't want to see a navigation build unless it includes the content. An optional view with the complete nav bar would be OK.
> This is mainly for reviewing PRs, especially ones that involve major content restructuring.

I see three options:
* Revert the PR and go back to the old UI/UX
* Create something that is for reviewing only (`curl --silent --show-error https://docs.theforeman.org/nightly/Provisioning_Hosts/index-katello.html | rg "^<h[1-9]"`)
* Make the tocify JS optional so that readers can enable/disable it themselves in the navigation.

**Comment by maximiliankolb** [comment]
I have no idea but I will ask around. This might take a while due to holidays etc.

**Comment by maximiliankolb** [comment]
@apinnick I spoke with a colleague about this. I had something like the dark mode toggle on https://docusaurus.io/ in my mind. It is possible but you might need to set a cookie (unverified!). Also, it will require some engineering to make the JS conditional based on user input.

**Comment by asteflova** [comment]
I agree that from the perspective of a reviewer, there might be valid concerns. But as mentioned here and in https://github.com/theforeman/foreman-documentation/pull/3453#issuecomment-2551809107, reviewers have a solid workaround available. Furthermore, they are not the target audience.

From the perspective of users, the change seems to be for the better. I'm also not aware of end users bringing up concerns regarding this change.

With that in mind, should we perhaps close the issue?

**Comment by apinnick** [comment]
If I'm the only one who dislikes this feature, I will close the issue.
